app/javascript/pages/IndividualEvents/query.gql

```diff
@@ -22,6 +22,24 @@ query IndividualEventsQuery {
         ...OfficeEntry
       }
     }
+    signups {
+      event {
+        id
```

Comment:
Is this field used?

Reply:
nope, you're right! It must've snuck in when I was testing, I'll remove this